### DISCUSSION TITLE
My monitor supports DDC/CI, but it doesn't change brightness automatically

### DISCUSSION THREAD

**Discussion by bonzibudd** [xanderfrangos/twinkle-tray#179] [Q&A]
I don't know if this is where I should post this, but I have been having trouble getting the monitor to change brightness automatically. I think it's probably impossible with my monitor, but just wanted to post it here in case there is a solution. Here is the information:
![image](https://user-images.githubusercontent.com/61938331/116728355-cf793780-a9b3-11eb-95ec-614cc95eeb09.png)

When I try to change the brightness, nothing happens. However, I am able to turn off the monitor, and when I turn it back on the brightness is changed.

**Comment by xanderfrangos** [comment]
Hi @bonzibudd,

I've been looking around to see if there's any reason for this to happen. It looks like the "Samsung Magic Bright" settings could override Twinkle Tray. If it's set to "Dynamic Contrast" (or possibly "Cinema"), that might be causing the problem. It could also be from brightness overrides in the AMD/NVIDIA/Intel control panel, if you've set those.

**Reply by bonzibudd** [reply]
Ok, thanks. I will look at some of those settings.
